src/tree/namespace/merkle_tree.cairo

```diff
@@ -27,22 +27,16 @@ struct NamespaceMerkleProof {
     key: u256,
     num_leaves: u256,
 }
-// mod NamespaceMerkleTree {
-//     use super::{Namespace, NamespaceNode, NamespaceMerkleMultiproof, NamespaceMerkleProof};
-//     use alexandria_bytes::BytesTrait;
-//     use blobstream_sn::tree::binary::hasher::leafDigest;
-
-//     fn verify(
-//         root: NamespaceNode,
-//         proof: NamespaceMerkleProof,
-//         namespace: Namespace,
-//         data: alexandria_bytes::bytes::Bytes
-//     ) -> bool {
-
-//         node : NamespaceNode = leafDigest(namespace, data);
-//         true
-//     }
-// }
+mod NamespaceMerkleTree {
```

Comment:
I don't think the above comment should be removed as part of this PR. Also, a lot of the `use` statements added are unneeded, I think.

Reply:
These are actually present as these are required for the merkle tree implementation